includes/class-wc-payments-apple-pay-registration.php

```diff
@@ -254,11 +254,33 @@ private function get_gateway_mode_string() {
 		return 'live';
 	}
 
+	/**
+	 * Check if merchant can register domain with Apple Pay.
+	 *
+	 * @return bool
+	 */
+	private function can_merchant_register_domain_with_applepay(): bool {
```

Comment:
May I suggest to move this code into the utilities file below? It is repeated here and above, and it'd be better to be in one place.